conf/rails

```diff
@@ -46,7 +46,25 @@ chmod +x /usr/local/rbenv/rbenv.d/rehash/gem-symlinks.bash
 [[ -n "$RBY_VER" ]] || echo "RBY_VER not set - falling back to latest stable Ruby"
 RBY_VER="${RBY_VER:--l}"
 
-MAKE_OPTS="-j$(nproc)" RUBY_CONFIGURE_OPTS=--disable-install-doc rbenv install $RBY_VER
+# ruby 2.x requires openssl 1.1 not available in bookworm
+# req: https://github.com/foodcoops/foodsoft/blob/master/.ruby-version
+if [[ "$RBY_VER" == 2.* ]] then
+  unset RELEASE
+  wget https://www.openssl.org/source/openssl-1.1.1n.tar.gz
```

Comment:
Is there any reason why you aren't using the latest v1.1.1? (v1.1.1w)

OpenSSL v1.1.1 is EOL, but at least that version doesn't currently have any [open CVEs](https://www.cvedetails.com/vulnerability-list/vendor_id-217/product_id-383/version_id-1657305/Openssl-Openssl-1.1.1w.html) for now. In fairness, a quick scan of the (6) [CVEs for v1.1.1t](https://www.cvedetails.com/vulnerability-list/vendor_id-217/product_id-383/version_id-1302028/Openssl-Openssl-1.1.1t.html) it appears that they aren't particularly problematic for our use case, it's still far from ideal.

Having said that, Foodsoft isn't a particularly popular appliance and TBH, I'm leaning towards putting it aside for v18.x.